src/box/errcode.h

```diff
@@ -300,23 +300,22 @@ struct errcode_record {
 	/*245 */_(ER_OLD_TERM,			"The term is outdated: old - %llu, new - %llu") \
 	/*246 */_(ER_INTERFERING_ELECTIONS,	"Interfering elections started")\
 	/*247 */_(ER_ITERATOR_POSITION,		"Iterator position is invalid") \
-	/*248 */_(ER_DDL_NOT_ALLOWED,		"DDL operations are not allowed: %s") \
-	/*249 */_(ER_UNKNOWN_AUTH_METHOD,	"Unknown authentication method '%s'") \
-	/*250 */_(ER_INVALID_AUTH_DATA,		"Invalid '%s' data: %s") \
-	/*251 */_(ER_INVALID_AUTH_REQUEST,	"Invalid '%s' request: %s") \
-	/*252 */_(ER_WEAK_PASSWORD,		"Password doesn't meet security requirements: %s") \
-	/*253 */_(ER_OLD_PASSWORD,		"Password must differ from last %d passwords") \
-	/*254 */_(ER_NO_SUCH_SESSION,		"Session %llu does not exist") \
-	/*255 */_(ER_WRONG_SESSION_TYPE,	"Session '%s' is not supported") \
-	/*256 */_(ER_PASSWORD_EXPIRED,		"Password expired") \
-	/*257 */_(ER_AUTH_DELAY,		"Too many authentication attempts") \
-	/*258 */_(ER_AUTH_REQUIRED,		"Authentication required") \
-	/*259 */_(ER_SQL_SEQ_SCAN,		"Scanning is not allowed for %s") \
-	/*260 */_(ER_NO_SUCH_EVENT,		"Unknown event %s") \
-	/*261 */_(ER_BOOTSTRAP_NOT_UNANIMOUS,	"Replica %s chose a different bootstrap leader %s") \
-	/*262 */_(ER_CANT_CHECK_BOOTSTRAP_LEADER, "Can't check who replica %s chose its bootstrap leader") \
-	/*263 */_(ER_BOOTSTRAP_CONNECTION_NOT_TO_ALL, "Some replica set members were not specified in box.cfg.replication") \
-	/*264 */_(ER_NIL_UUID,			"Nil UUID is reserved and can't be used in replication") \
+	/*248 */_(ER_UNKNOWN_AUTH_METHOD,	"Unknown authentication method '%s'") \
+	/*249 */_(ER_INVALID_AUTH_DATA,		"Invalid '%s' data: %s") \
+	/*250 */_(ER_INVALID_AUTH_REQUEST,	"Invalid '%s' request: %s") \
+	/*251 */_(ER_WEAK_PASSWORD,		"Password doesn't meet security requirements: %s") \
+	/*252 */_(ER_OLD_PASSWORD,		"Password must differ from last %d passwords") \
+	/*253 */_(ER_NO_SUCH_SESSION,		"Session %llu does not exist") \
+	/*254 */_(ER_WRONG_SESSION_TYPE,	"Session '%s' is not supported") \
+	/*255 */_(ER_PASSWORD_EXPIRED,		"Password expired") \
+	/*256 */_(ER_AUTH_DELAY,		"Too many authentication attempts") \
+	/*257 */_(ER_AUTH_REQUIRED,		"Authentication required") \
+	/*258 */_(ER_SQL_SEQ_SCAN,		"Scanning is not allowed for %s") \
+	/*259 */_(ER_NO_SUCH_EVENT,		"Unknown event %s") \
+	/*260 */_(ER_BOOTSTRAP_NOT_UNANIMOUS,	"Replica %s chose a different bootstrap leader %s") \
+	/*261 */_(ER_CANT_CHECK_BOOTSTRAP_LEADER, "Can't check who replica %s chose its bootstrap leader") \
+	/*262 */_(ER_BOOTSTRAP_CONNECTION_NOT_TO_ALL, "Some replica set members were not specified in box.cfg.replication") \
+	/*263 */_(ER_NIL_UUID,			"Nil UUID is reserved and can't be used in replication") \
```

Comment:
Please don't move all these error codes. Instead replace `ER_DDL_NOT_ALLOWED` with `ER_UNUSED` like it was done in [this commit](https://github.com/tarantool/tarantool/commit/d026b546fa6b437c90c6c7849e3a0dfe9ae76f92#diff-3ab8a0e9527079f220221deb0f67e18ea14b35d814ab2300055745626e7b314f) for example. Later we'll reuse it for something else.

Reply:
Done.